### ISSUE TITLE
Alignment of calendar days are incorrect on Agenda View

### ISSUE THREAD

**Issue by Nealm03** [wix/react-native-calendars#690]
Hi, I have an issue that I can't seem to resolve. I don't think the issue is directly coming from this package but it is affecting it quite adversely. I have tried process of elimination, stack overflow question and testing but can't seem to fix it.
Any help whatsoever would be greatly appreciated. I have also tried to wrap the Agenda inside a <View> messing around with each justifyContent setting. No configuration seems to work
## Description

Short summary: it seems that some type of justifyContent is throwing off alignment of the app. I'm not entirely sure what the origin of the issue is, however it affects the calendar in a minimal repro. 

## Expected Behavior

Days to align under corresponding number, and onPress to fire on pressed date.

## Observed Behavior

When touching to the far left, the element inspector highlights invisible element on the far right.
Please see my SO question: https://stackoverflow.com/questions/53500637/react-native-calendars-wix-agenda-view-alignment-broken

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-calendars`: react-native-calendars@1.2.1
* `npm ls react-native`: react-native@0.57.1

Also specify:

1. IOS 12.1 {IPhone 5s} / Expo / sdk version 31

## Reproducible Demo

https://github.com/FatOblivion/ReactNativeWixRepro

**Comment by Nealm03** [comment]
Apologies for opening this issue on this repro, It was quite an issue to track down exactly the origin of the issue. It appears that this was an expo issue itself. 
TL;DR: uninstall expo app, reinstall then run `expo start --clear` and the alignment is fixed. 

Again, apologies to anyone who took any time looking into this issue
